packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -60,11 +61,13 @@ function App(): JSX.Element {
           onRenderTypingIndicator={
             customDataModel
               ? (typingUsers) => (
-                  <text id="custom-data-model-typing-indicator">
-                    {typingUsers.length > 0
-                      ? `${typingUsers.map((user) => user.displayName).join(',')} is typing...`.toUpperCase()
-                      : 'No one is currently typing.'}
-                  </text>
+                  <Stack style={{ width: '100%' }}>
```

Comment:
Without this does it align to the center? If so, for developers might be nicer if we left align by default (if the change is trivial)